recipe/conda_build_config.yaml

```diff
@@ -17,4 +21,4 @@ pin_run_as_build:
     max_pin: x.x
   suitesparse:
     max_pin: x.x  # [not osx]
-    max_pin: x.x.x  # [osx]
\ No newline at end of file
+    max_pin: x.x.x  # [osx]
```

Comment:
It is really annoying that the update bot removes EOF newlines...

Reply:
I think I have a fix for that: https://github.com/regro/cf-scripts/pull/2696